<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoodMenu</class>
 <widget class="QDialog" name="FoodMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Segoe UI&quot;;
color: rgb(170, 170, 127);</string>
   </property>
   <property name="text">
    <string>Chicken Options</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Segoe UI&quot;;
color: rgb(85, 0, 0);</string>
   </property>
   <property name="text">
    <string>Meat Options</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Chicken_options">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select from Chicken</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fried Chicken</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grilled Chicken</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chicken Sandwich</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Meat_options">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select from Meat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Steak</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Burger</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Meatball Pasta</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Account Balance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Balance">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Iteam Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(7, 189, 255);
color: rgb(85, 0, 127);</string>
   </property>
   <property name="text">
    <string>Add to cart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(197, 228, 255);</string>
   </property>
   <property name="text">
    <string>View Cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of added items:</string>
   </property>
  </widget>
  <widget class="QLabel" name="iteams">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Alert">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>Alert</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
